doc/getting_started/index.rst

```diff
@@ -0,0 +1,55 @@
+***************
+Getting started
+***************
+
+
+Installation
+------------
+
+HoloPlot supports Python 2.7-3.7 on Linux, Windows, or Mac.
+
+Installable ``holoplot`` packages will be created at some point, but
+at present, the recommended way to install HoloPlot is based on conda
+`conda <http://conda.pydata.org/docs>`_ environments and 
+`git <https://git-scm.com>`_:
+
```

Comment:
Presumably this needs updating as soon as you release a package.

Reply:
Yes, I'll update it shortly.